Proposals/TypeNetwork_ParametricAxes/ProposalSummary_ytlc.md

```diff
@@ -50,6 +50,8 @@ calling this axis "x height," we see conventionality benefits in a name that fit
 systematic structure of the overall system of proposed axes: Y dimension transparency of
 lowercase letters.
 
+ytlc is measured from the baseline to the top x-height
```

Comment:
Since this is part of the portion that would be published in the registry, can you please formulate as a full, punctuated sentence; e.g.,

"The ytlc value is measured from the baseline to the top of the x-height.

Also, wouldn't this make sense to be added to the Scale Interpretation field, rather than Additional Information?